features/safe-area-inset.yml

```diff
@@ -2,6 +2,8 @@ name: Safe area inset environment variables
 description: The `safe-area-inset-` CSS environment variables represent a rectangle that cannot cut off content within a non-rectangular screen. For example, `top:\ env(safe-area-inset-top);` positions the top of an element below the notch on an iPhone.
 spec: https://drafts.csswg.org/css-env-1/#safe-area-insets
 group: environment-variables
+# Caniuse is for env(), but this currently is the only supported value.
```

Comment:
There's also titlebar-area-x/y/width/height, so safe-area-inset isn't the only variable that's supported in env(). I believe there may also be some virtual keyboard variables too.

Reply:
Good point. I misremembered the support on title-bar-area-*. I still think this is a valid mapping since it was the first one (and I updated the comment).